cmd/whereabouts_test.go

```diff
@@ -557,12 +557,16 @@ var _ = Describe("Whereabouts operations", func() {
 			Netns:       nspath,
 			IfName:      ifname,
 			StdinData:   []byte(conf),
-			Args:        "IgnoreUnknown=1;K8S_POD_NAMESPACE=dummyNS;K8S_POD_NAME=dummyPOD",
+			Args:        cniArgs(podNamespace, podName),
```

Comment:
I don't understand this commit at all. This code already exists in the master branch. You should rebase your code instead of updating this manually.

Reply:
It is part of the new test introduced in this PR so it didn't exist in the master branch.

https://github.com/k8snetworkplumbingwg/whereabouts/blob/310ad3d8ae10e8d199d6c62aaa23c21d16f223b0/cmd/whereabouts_test.go#L534

This was added later on while updating the tests based on new changes merged in the master branch.
It will be squashed with the commit of this test :)